Extension/src/LanguageServer/client.ts

```diff
@@ -620,6 +627,16 @@ class DefaultClient implements Client {
         this.model.tagParserStatus.Value = notificationBody.status;
     }
 
+    private updateInactiveRegions(params: InactiveRegionParams): void {
+        let renderOptions: vscode.DecorationRenderOptions = {
+            color: "rgba(255,0,255,0.5)"
+        };
+        let textEditorDecoration = vscode.window.createTextEditorDecorationType(renderOptions);
+        
+        let editor = vscode.window.visibleTextEditors.find(e => e.document.uri.toString() == params.uri); // TODO change to document.uri == params.uri
```

Comment:
What does this TODO mean?

Reply:
This is a TODO that I should have investigated earlier. Most operations within client.ts operate via vscode.Uri, whereas this method only compares the absolute file path via string. @bobbrow opinion on the matter?

Reply:
You should probably send the uri in the JSON message instead of the filename.